tools/ci-build-other-configs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# This is expected to be run from the toplevel of the OCaml source tree
+# This script can be removed once none of the branch tested on Inria's CI
+# expects to find it here, i.e. once we stop working on the 4.07 branch
+./tools/ci/inria/other-configs $*
```

Comment:
I's rather use `"$@"` instead of `$*` in the off chance that there are spaces in the arguments.